### PR TITLE
refactor: add new class Leetcode in api to manage daily posts

### PR DESCRIPTION
**Pull request by vetom**
In this PR, I'm creating the Leetcode class for handling the daily and weekly challenge posts. This PR might be related to #40, because I'm using the same discord client for both classes.

### CODE REVIEW

**Comment by kajahno** [comment]
@vetom was this tested?

**Comment by vetom** [comment]
@kajahno Not yet. I need some help setting the local env 🙏

File: api/leetcode.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import * as leetcode from "./leetcode";
```
[the remark this answers is not in the document]
Comment (vetom) [reply api/leetcode.js]: Updated! 😁

**Review by kajahno — APPROVED** [review]
Thanks for this! We've paired so def LGTM

**Pull request merged**